.env.sample

```diff
@@ -1,4 +1,3 @@
-FX_REMOTE_SETTINGS_WRITER_SERVER="https://settings-writer.prod.mozaws.net/v1"
-FX_REMOTE_SETTINGS_WRITER_USER=""
-FX_REMOTE_SETTINGS_WRITER_PASS=""
-NODE_ENV="production"
+SERVER="https://remote-settings.allizom.org/v1"
+AUTHORIZATION="foo:bar"
+NODE_ENV="stage"
```

Comment:
`NODE_ENV` is not used in the script. 
It is usually used in frameworks for log level etc. but not here. We can just remove it. 

We could have an `ENVIRONMENT` variable, and do different things whether we're on DEV or STAGE/PROD (eg. approve own changes), but maybe out of scope here.